eventmessages.civix.php

```diff
@@ -41,7 +41,7 @@ public static function ts($text, $params = []): string {
    *   Ex: 'http://example.org/sites/default/ext/org.example.foo'.
    *   Ex: 'http://example.org/sites/default/ext/org.example.foo/css/foo.css'.
    */
-  public static function url($file = NULL): string {
+  public static function url($file = NULL) {
```

Comment:
Not necessary. Seems that the same holds true for the other changes in this file. If you think this file should be updated, do this in a separate PR, please.